tensorboard/webapp/notification_center/_views/notification_center_test.ts

```diff
@@ -33,7 +50,14 @@ describe('notification center', () => {
         NoopAnimationsModule,
       ],
       declarations: [NotificationCenterContainer, NotificationCenterComponent],
+      providers: [
+        provideMockStore({
+          initialState: [],
+        }),
+      ],
     }).compileComponents();
+    store = TestBed.inject<Store<State>>(Store) as MockStore<State>;
+    store.overrideSelector(selectors.getNotifications, notificationNotes);
```

Comment:
remove this line here and move it to L64. 

By reading spec inside `it` in L64, you should be able to tell whether your assertion is correct. So, instead, there, 
```ts
// minimal data structure to test the module.
store.overrideSelector(selectors.getNotifications, [
 {
    category: CategoryEnum.WHATS_NEW,
    dateInMs: 100,
    title: '2.4 release',
    content: 'some testable content',
  },
]);
```

Reply:
Thanks for the suggestion! Addressed.